scripts/android/tests/HelloCodenameOneInstrumentedTest.java

```diff
@@ -379,4 +384,78 @@ public void testBrowserComponentScreenshot() throws Exception {
 
         emitScreenshot(capture, BROWSER_TEST);
     }
+
+    @Test
+    public void testInPlaceEditViewNPE() throws Exception {
+        Context ctx = ApplicationProvider.getApplicationContext();
+        try (ActivityScenario<Activity> scenario = launchMainActivity(ctx)) {
+            final CountDownLatch latch = new CountDownLatch(1);
+            final Throwable[] error = new Throwable[1];
+
+            scenario.onActivity(activity -> Display.getInstance().callSerially(() -> {
+                try {
+                    if (!(Display.getInstance().getImplementation() instanceof AndroidImplementation)) {
+                        println("CN1SS:WARN: Not running on AndroidImplementation, skipping InPlaceEditView test");
```

Comment:
This should fail. Since we're on Android `AndroidImplementation` must always be our implementation.